src/lambda/release.js

```diff
@@ -11,7 +11,7 @@ export default async function releaseLambda(req, _context) {
 		status: 200,
 		headers: {
 			'Content-Type': 'application/json',
-			'Cache-Control': 'public, max-age=3600, stale-if-error=86400'
+			'Cache-Control': 'public, max-age=86400, stale-if-error=86400'
```

Comment:
Bumped the cache age here as we're not releasing anywhere near often enough for an hour-long cache to really be justified.